src/java/org/apache/cassandra/schema/KeyspaceParams.java

```diff
@@ -94,6 +109,25 @@ public void validate(String name)
         replication.validate(name);
     }
 
+    /**
+     * Used to pick the default replication strategy for all distributed system keyspaces.
+     * The default will be SimpleStrategy and a hard coded RF factor.
+     * <p>
+     * One can change this default to NTS by passing in system properties:
+     * -Dcassandra.system_distributed_replication_per_dc=3
+     * -Dcassandra.system_distributed_replication_dc_names=cloud-east,cloud-west
+     */
+    public static KeyspaceParams systemDistributed(int rf)
+    {
+        if (!SYSTEM_DISTRIBUTED_NTS_OVERRIDE.isEmpty())
+        {
+            logger.info("Using override for distributed system keyspaces: {}", SYSTEM_DISTRIBUTED_NTS_OVERRIDE);
```

Comment:
I would move this into getSystemDistributedNtsOverride so that it does not print out this same message every time this is called.  It doesn't have the key space name or anything in the message, so it is not useful to print more than once.

Reply:
this message logically makes more sense in this place (it will be printed only 3 times), because this is the place where we really override something. I'll move it back to the `getSystemDistributedNtsOverride` method if you insist though

Reply:
I don't feel super strong.  But I do think it causes noise in the logs to print it extra times with no extra relevant information in the messages about why it comes out multiple times.